src/style/palette.stories.jsx

```diff
@@ -0,0 +1,183 @@
+import { Title, Subtitle, Description, Stories } from '@storybook/addon-docs'
+import PropTypes from 'prop-types'
+import styles from './palette.stories.module.scss'
+import paletteScss from './palette.scss'
+import { parseScss } from '../utils/parse'
+
+const paletteVariables = parseScss(paletteScss)
+
+const LABELS = {
+  primary: 'Primary',
+  secondary: 'Secondary',
+  tertiary: 'Tertiary',
+  success: 'Success',
+  danger: 'Danger',
+  warning: 'Warning',
+  'primary-hover': 'Primary Hover',
+  'secondary-hover': 'Secondary Hover',
+  'tertiary-hover': 'Tertiary Hover',
+  'success-hover': 'Success Hover',
+  'danger-hover': 'Danger Hover',
+  'warning-hover': 'Warning Hover',
+  'primary-active': 'Primary Active',
+  'secondary-active': 'Secondary Active',
+  'tertiary-active': 'Tertiary Active',
+  'success-active': 'Success Active',
+  'danger-active': 'Danger Active',
+  'warning-active': 'Warning Active',
+  darkindigo: 'DarkIndigo',
+  blackish: 'Blackish',
+  navy: 'Navy',
+  'american-gray': 'AmericanGray',
+  aquamarine: 'Aquamarine',
+  explosive: 'Explosive',
+  green: 'Green',
+  winter: 'Winter',
+  'grass-green': 'GrassGreen',
+  river: 'River',
+  'bright-green': 'BrightGreen',
+  darkblue: 'Darkblue',
+  saladish: 'Saladish',
+  lightblue: 'Lightblue',
+  'egg-yolk': 'EggYolk',
+  chillblue: 'Chillblue',
+  orange: 'Orange',
+  indigo: 'Indigo',
+  peach: 'Peach',
+  darkpurple: 'Darkpurple',
+  sunset: 'Sunset',
+  berry: 'Berry',
+  'dark-orange': 'DarkOrange',
+  purple: 'Purple',
+  red: 'Red',
+  'sofia-pink': 'SofiaPink',
+  'dark-red': 'DarkRed',
+  lipstick: 'Lipstick',
+  brown: 'Brown',
+  bubble: 'Bubble',
+  dark: 'Dark',
+  'wolf-grey': 'WolfGrey',
+  'ui-grey': 'UIGrey',
+  riverstone: 'Riverstone',
+  white: 'White'
+}
+
+const Color = ({ name, darkContrast, position }) => {
```

Comment:
Why is this here?
Below you have the main colors and the color coding, not really sure why this is here

Reply:
That is the component that renders each color strip with it's name and color code, it's used below

Reply:
I don't think this is bad, but since the storybook is the documentation, I would tell you not to waste time doing components that are only used here

Reply:
I think I did it to **save** time, otherwise I would have repeated this 7 line component in each one of the 53 colors 😄 
I understand the point tho, thanks for the comments!